server/app/lib/actions/fusor/deployment/open_shift/setup_ose.rb

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+
+require "#{Fusor::Engine.root}/lib/modules/ose-installer/launch.rb"
```

Comment:
Change it to ```ose_installer```. I had a similar problem when I copied the SSHConnection from egon. In egon it was named ```ssh-connection```. I had to rename it to ```ssh_connection```. So rename it to ```ose_installer``` and it will work. That's why it's not loading when running in foreman tasks.